src/picai_prep/dcm2mha.py

```diff
@@ -187,25 +195,26 @@ def _extract_metadata(self):
                     item['error'] = f'Unexpected error: {e}'
                     self.item_log(item, 'unexpected error')
                 else:
-                    total += len(item['dcms'])
+                    total += (len(item['dcms']) if 'dcms' in item else 0)
 
         self.info(f"Collected {plural(total, 'DICOM file')} from {self.valid_items_str()}.", self.get_history_report())
 
+    @staticmethod
```

Comment:
Why? Only one usage.

Reply:
Stacked functions make it difficult to follow what's going on, so it there is no benefit, better to keep it simple.